plugins/woocommerce/client/blocks/assets/js/editor-components/product-control/index.tsx

```diff
@@ -46,6 +46,14 @@ interface ProductControlProps {
 	 * The ID of the currently expanded product.
 	 */
 	expandedProduct: number | null;
+	/**
+	 * Callback to load more variations.
+	 */
+	onLoadMore: () => void;
```

Comment:
`onLoadMore()` sounds like it loads more products rather than more variations for the current `expandedProduct`.